lib/controllers/BaseController.js

```diff
@@ -109,6 +108,22 @@ class BaseController {
   getPlan(plan_id) {
     return catalog.getPlan(plan_id);
   }
+  getSpaceGuid(deploymentName) {
+    return this.cloudController.getSpaceGuid(this.getInstanceId(deploymentName));
+  }
+  getOrgGuid(deploymentName) {
+    return this.cloudController.getOrgGuid(this.getInstanceId(deploymentName));
+  }
+  createOpts(deploymentName) {
```

Comment:
rename this method to `getOrgAndSpaceGuid` its better than having generic names like this as it helps with readability when one just reads this method from the subclass. Else from subclass he needs to jump into the method definition to know what this createOpts is all about.